src/GeositeFramework/js/Plugin.js

```diff
@@ -668,8 +668,13 @@ require(['use!Geosite',
                 var pluginModel = this.model,
                     pluginObject = pluginModel.get('pluginObject');
 
-                var img = $('<img class="graphic" />').attr('src', pluginObject.infoGraphic);
-                this.$el.append(img);
+	       if (pluginObject.infoGraphic.slice(pluginObject.infoGraphic.length - 4, pluginObject.infoGraphic.length) == ".jpg" || pluginObject.infoGraphic.slice(pluginObject.infoGraphic.length - 4, pluginObject.infoGraphic.length) == ".png") {
+		var snippit = $('<img class="graphic" />').attr('src', pluginObject.infoGraphic);
+	       } else {
+		var snippit = $(pluginObject.infoGraphic);
+	       }				
```

Comment:
It looks like a mix of spaces and tabs are used for indentation here. Consider using all-spaces for consistency.